### ISSUE TITLE
"wrangler preview --watch" error 

### ISSUE THREAD

**Issue by toinbis** [cloudflare/rustwasm-worker-template#10]
Hi,

I generate wrangler project with --type=rust, modified wrangler.toml accordingly and tried to launch:
`wrangler preview --watch` just to receive this output:

```
🌀  Compiling your project to WebAssembly...
[INFO]: 🎯  Checking for the Wasm target...
[INFO]: 🌀  Compiling to Wasm...
    Finished release [optimized] target(s) in 0.06s
[INFO]: ⬇️  Installing wasm-bindgen...
[INFO]: Optional fields missing from Cargo.toml: 'description', 'repository', and 'license'. These are not necessary, but recommended
[INFO]: ✨   Done in 0.20s
[INFO]: 📦   Your wasm pkg is ready to publish at ./pkg.
Error: Your JavaScript project is missing a `package.json` file; is `./` the wrong directory?
```

What am I missing? Why is it looking for package.json if it’s a rustwasm project?

Thanks!

**Comment by toinbis** [comment]
@ashleygwilliams  Would you be able to look into this? No rush. Thanks!

**Comment by toinbis** [comment]
Seems to be fixed in wrangler@1.3.1.
